.circleci/config.yml

```diff
@@ -1,113 +0,0 @@
-            then
-              export HOME=$(getent passwd $(id -un) | cut -d: -f6)
-            fi
-
-            mkdir -p ~/.ssh
-
-            echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==
-            bitbucket.org ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAubiN81eDcafrgMeLzaFPsw2kNvEcqTKl/VqLat/MaB33pZy0y3rJZtnqwR2qOOvbwKZYKiEO1O6VqNEBxKvJJelCq0dTXWT5pbO2gDXC6h6QDXCaHo6pOHGPUy+YBaGQRGuSusMEASYiWunYN0vCAI8QaXnWMXNMdFP3jHAJH0eDsoiGnLPBlBp4TNm6rYI74nMzgz3B9IikW4WVK+dc8KZJZWYjAuORU3jc1c/NPskD2ASinf8v3xnfXeukU0sJ5N6m5E8VLjObPEO+mN2t/FZTMZLiFqPWc/ALSqnMnnhwrNi2rbfg/rd/IpL8Le3pSBne8+seeFVBoGqzHM9yXw==
-            ' >> ~/.ssh/known_hosts
-
-            (umask 077; touch ~/.ssh/id_rsa)
-            chmod 0600 ~/.ssh/id_rsa
-            (cat <<EOF > ~/.ssh/id_rsa
-            $CHECKOUT_KEY
-            EOF
-            )
-
-            # use git+ssh instead of https
-            git config --global url."ssh://git@github.com".insteadOf "https://github.com" || true
-
-            # MODIFICATIONS
-            if [ -n "$CIRCLE_TAG" ]
-            then
-              echo "Tags not supported"
-              exit 1
-            fi
-
-            if [ -e /home/circleci/project/.git ]
-            then
-              echo "Option not supported. You can't checkout from an cached repo"
-              exit 2
-            fi
-
-
-            mkdir -p /home/circleci/project
-            cd /home/circleci/project
-            git clone "$CIRCLE_REPOSITORY_URL" .
-            if [ "CIRCLE_BRANCH" == "master" ]
-            then
-              git fetch --force origin "master:remotes/origin/master"
-            fi
-
-            if [ -n "$CIRCLE_BRANCH" ]
-            then
-              git reset --hard "$CIRCLE_SHA1"
-              git checkout -q -B "$CIRCLE_BRANCH"
-            else
-              git fetch --force origin "${CIRCLE_BRANCH}/head:remotes/origin/${CIRCLE_BRANCH}"
-            fi
-
-            git reset --hard "$CIRCLE_SHA1"
-      - run:
-          name: Determine which projects have changed and trigger the builds
-          command: |
-            ## Identify head of master
-            git ls-remote origin | awk "/master/ {print \$1}" > master
-            MASTER_COMMIT=`cat master`
-
-            #Determine range of SHAs for diffing files
-            COMMITS="${CIRCLE_SHA1}..${MASTER_COMMIT}"
-
-            #Check the diff on the commits
-            git diff --name-only $COMMITS | cut -d/ -f1 | sort -u > projects
-            echo -e "Modified directories:\n`cat projects`\n"
-
-            # If modified directories contain yarn, buildkite, or circleci directories
-            buildall=0
```

Comment:
are we good to kill this logic? Do we have a similar step in buildkite?

Reply:
circleCI doesn't pass anymore, regardless. We'd have to do the same thing in buildkite if we still care about this logic

Reply:
I see, didn't notice that Circle is not doing anything before. All good then 👍